### DISCUSSION TITLE
Faster_whisper does not work due to ctranslate2 problem

### DISCUSSION THREAD

**Discussion by VincentRingstrom** [SYSTRAN/faster-whisper#231] [General]
Hello I get this error when running faster-whisper. 

How can this be fixed?

Apparently, StorageView is not an attribute of ctranslate2. And this is the line where the error occurs in faster-whisper:

 def generate_with_fallback(
        self,
        encoder_output: ctranslate2.StorageView,
        prompt: List[int],
        tokenizer: Tokenizer,
        options: TranscriptionOptions,
    ) -> Tuple[ctranslate2.models.WhisperGenerationResult, float, float]:

  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\faster_whisper\__init__.py", line 2, in <module>
    from faster_whisper.transcribe import WhisperModel
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\faster_whisper\transcribe.py", line 58, in <module>
    class WhisperModel:
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\faster_whisper\transcribe.py", line 444, in WhisperModel
    encoder_output: ctranslate2.StorageView,
AttributeError: module 'ctranslate2' has no attribute 'StorageView'

**Comment by guillaumekln** [comment]
How did you install faster-whisper?

Also make sure you don’t have a file named `ctranslate2.py`.

**Reply by VincentRingstrom** [reply]
Aha I found my problem. I re-installed faster-whisper with --force-reinstall. But when I look at my pip errors, in the last days I have installed libreTranslate and argostranslate to translate from whisper(since whisper is bad at translation can only handle transcription well). 
Argostranslate changed the ctranslate2 version without notifying me beforehand into an unsupported version for this library.

Thank you for your quick response





ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.
libretranslate 1.3.10 requires requests==2.28.1, but you have requests 2.30.0 which is incompatible.
argostranslate 1.8.0 requires ctranslate2==2.24.0, but you have ctranslate2 3.13.0 which is incompatible.
